src/OSPSuite.Core/Qualification/QualifcationConfiguration.cs

```diff
@@ -36,6 +36,12 @@ public class QualifcationConfiguration : IValidatable
       /// </summary>
       public string MappingFile { get; set; }
 
+      /// <summary>
+      ///    Path of log file that will be created for the qualification runner.
+      ///    and will include all entries from all qualification processes run.
+      /// </summary>
+      public string QRSharedLogPath { get; set; }
```

Comment:
I don't understand why we need this? Log files are passed as parameter to the cli...

Reply:
you are right, I was copying it here without reason. removed